doc/changelog.md

```diff
@@ -10,6 +10,8 @@
 * Adds fallback compiler drivers to increase reliability during linking phase. Also adds support for a
   CATALYST_CC environment variable for manual specification of the compiler driver used for linking.
   [#30](https://github.com/PennyLaneAI/catalyst/pull/30)
+* Use Makefiles for CI/CD.
+  [#41](https://github.com/PennyLaneAI/catalyst/pull/41)

```

Comment:
We're not adding repo changes to the changelog and only package changes should be added here. So, I think we can remove this,
```suggestion
```